src/eosjs-key-conversions.ts

```diff
@@ -18,7 +18,8 @@ export const constructElliptic = (type: KeyType): EC => {
 
 export const generateKeyPair = (type: KeyType, options?: EC.GenKeyPairOptions):
     {publicKey: PublicKey, privateKey: PrivateKey} => {
-    if (process.env.EOSJS_KEYGEN_ALLOWED !== 'true') {
+    // @ts-ignore
+    if (process.env.EOSJS_KEYGEN_ALLOWED !== 'true' && window.EOSJS_KEYGEN_ALLOWED !== 'true') {
         throw new Error('Key generation is completely INSECURE in production environments in the browser. ' +
```

Comment:
Should this error message be updated to mention setting `EOSJS_KEYGEN_ALLOWED` on the window?

Reply:
Updated!